streams-util/src/main/java/org/apache/streams/util/SerializationUtil.java

```diff
@@ -49,7 +49,7 @@
      * @param serialized
      * @return
      */
-    public static Object deserialize(byte[] serialized, Class clazz) {
+    public static Object deserialize(byte[] serialized) {
```

Comment:
Typically yes, but this is actually reverting to what it was in 0.2-incubating release.